src/acquisition/covidcast/signal_dash_data_generator.py

```diff
@@ -17,7 +18,9 @@
 from delphi.epidata.acquisition.covidcast.logger import get_structured_logger
 
 
-LOOKBACK_DAYS_FOR_COVERAGE = 28
+LOOKBACK_DAYS_FOR_COVERAGE = 52
```

Comment:
out of curiousity, why 52?

Reply:
uhhh because I can't multiple 28 by 2, apparently